apps/node/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "api-example",
+  "version": "1.0.0",
+  "description": "A demo application to demonstrate Attentive's oAuth, API and webhooks",
+  "main": "server.js",
+  "scripts": {
+    "test": "test",
+    "start": "node server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/attentive-mobile/api-examples.git"
+  },
+  "keywords": [
+    "attentive",
+    "sms"
+  ],
+  "author": "Public API",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/attentive-mobile/api-examples/issues"
+  },
+  "homepage": "https://github.com/attentive-mobile/api-examples#readme",
+  "dependencies": {
+    "axios": "^0.24.0",
+    "crypto": "^1.0.1",
```

Comment:
You don't need this dependency, `crypto` has been built into node for a long time.